lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -179,6 +246,108 @@ class DetailsRenderer {
     pre.textContent = details.text;
     return pre;
   }
+
+  /**
+   * @param {!DetailsRenderer.CRCDetailsJSON} details
+   * @return {!Node}
+   */
+  _renderCriticalRequestChains(details) {
+    const tmpl = this._dom.cloneTemplate('#tmpl-lh-crc', this._templateContext);
+
+    // Fill in top summary.
+    this._dom.find('.lh-crc__longest_duration', tmpl).textContent =
+        Util.formatNumber(details.longestChain.duration) + 'ms';
+    this._dom.find('.lh-crc__longest_length', tmpl).textContent = details.longestChain.length;
+    this._dom.find('.lh-crc__longest_transfersize', tmpl).textContent =
+        Util.formateBytesToKB(details.longestChain.transferSize) + 'KB';
+
+    const detailsEl = this._dom.find('.lh-details', tmpl);
+
+    this._dom.find(':scope > summary', detailsEl).textContent = details.header.text;
+
+    /**
+     * Creates the DOM for a tree segment.
+     * @param {!DetailsRenderer.CRCSegment} segment
+     * @return {!Node}
+     */
+    const createChainNode = segment => {
+      const chainsEl = this._dom.cloneTemplate('#tmpl-lh-crc__chains', tmpl);
+
+      // Hovering over request shows full URL.
+      this._dom.find('.crc-node', chainsEl).setAttribute('title', segment.node.request.url);
+
+      const treeMarkeEl = this._dom.find('.crc-node__tree-marker', chainsEl);
+
+      // Construct lines and add spacers for sub requests.
```

Comment:
This logic follows critical-request-chains.html fairly closely. Compare if you wish.